### DISCUSSION TITLE
Xiaomi Redmi Note 9 Pro screen server

### DISCUSSION THREAD

**Discussion by hanbozdugan** [NightscoutFoundation/xDrip#3096] [General]
Hello. I have a Xiaomi Redmi Note 9 Pro phone which does not have the screen server function and on lock screen xDrip+ only gives me the sensor value without any other additional graphics as shown in the first picture. But I found that when the bluetooth connection with the sensor is lost, this graphic is displayed on the same phone on lock screen as shown in the second picture. I have installed the latest version of xDrip+ from 09/23/2023 as shown in the third picture. My request is how on this phone in normal operation and connection to the sensor, how to get the xDrip+ graphic with locked display, where it is visible in the second picture when the signal is lost.
![1695541271071](https://github.com/NightscoutFoundation/xDrip/assets/8653097/ace1ff7f-ae23-4984-9604-7b25f4e48831)
![1695541095549](https://github.com/NightscoutFoundation/xDrip/assets/8653097/0e9a2919-d885-41d5-87b0-f9961fe36203)
![1695541566878](https://github.com/NightscoutFoundation/xDrip/assets/8653097/4f06e570-64b2-42e9-ac0b-8968ef304e71)

**Comment by Navid200** [comment]
Was this resolved?
If yes, please explain how so that others who may find this through search can see the solution also.
Thanks

**Reply by hanbozdugan** [reply]
This is the solution
![1695601560046](https://github.com/NightscoutFoundation/xDrip/assets/8653097/497cc4c6-a3e6-4b92-91cf-2154a8313906)
![1695580883762](https://github.com/NightscoutFoundation/xDrip/assets/8653097/16d9619c-b374-4191-ac2e-f8e54f3a9c9e)
